experimental/dds/tree2/src/feature-libraries/defaultFieldChangeCodecs.ts

```diff
@@ -7,66 +7,20 @@ import { TAnySchema, Type } from "@sinclair/typebox";
 import { ICodecFamily, IJsonCodec, makeCodecFamily, makeValueCodec, unitCodec } from "../codec";
 import { JsonCompatibleReadOnly, Mutable } from "../util";
 import { jsonableTreeFromCursor, singleTextCursor } from "./treeTextCursor";
-import type {
-	NodeUpdate,
-	OptionalChangeset,
-	OptionalFieldChange,
-	ValueChangeset,
-} from "./defaultFieldChangeTypes";
+import type { NodeUpdate, OptionalChangeset, OptionalFieldChange } from "./defaultFieldChangeTypes";
 import type { NodeChangeset } from "./modular-schema";
-import {
-	EncodedValueChangeset,
-	EncodedOptionalChangeset,
-	EncodedNodeUpdate,
-} from "./defaultFieldChangeFormat";
+import { EncodedOptionalChangeset, EncodedNodeUpdate } from "./defaultFieldChangeFormat";
 
 export const noChangeCodecFamily: ICodecFamily<0> = makeCodecFamily([[0, unitCodec]]);
 
 export const counterCodecFamily: ICodecFamily<number> = makeCodecFamily([
 	[0, makeValueCodec(Type.Number())],
 ]);
 
-export const makeValueFieldCodecFamily = (childCodec: IJsonCodec<NodeChangeset>) =>
-	makeCodecFamily([[0, makeValueFieldCodec(childCodec)]]);
-
 export const makeOptionalFieldCodecFamily = (
 	childCodec: IJsonCodec<NodeChangeset>,
 ): ICodecFamily<OptionalChangeset> => makeCodecFamily([[0, makeOptionalFieldCodec(childCodec)]]);
 
-function makeValueFieldCodec(
```

Comment:
You can also delete `EncodedValueChangeset` in defaultFieldChangeFormat.ts and `ValueChangeset` in defaultFieldChangeTypes.ts

Reply:
Deleted in latest commit.